thirdparty/drmingw_build_mingw.cmd

```diff
@@ -81,6 +83,9 @@ goto :eof
 rem Download !URL! to !DEST!.
 rem Note: Positional parameters don't work here, as the percent signs in URLs are eagerly interpreted despite quoting.
 
+@echo Downloading %URL%
+@echo   %DEST%
+

```

Comment:
This batch file uses delayed expansion, so for consistency this should use that syntax.

Reply:
Why use it if it's not needed? That just suggests a misunderstanding of cmd/batch.

Reply:
It was needed somewhere, so we might as well use it everywhere for consistency. Is there a reason not to?

Reply:
Using it ubiquitously seems like voodoo programming to me.
There are actually some occasions when you _don't_ want the expansion to be delayed, but this isn't one of them.
I can't think of a functional reason to avoid it other than suggesting that you don't know how the "feature" works. I only use it when it's necessary (e.g. inside parens for `FOR` or `IF`).
I prefer to be precise about the use of features, but since I didn't go back and "fix" all the other instances of it, I can at least imagine that it could be confusing to other developers.